<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAlarm</class>
 <widget class="QDialog" name="AddAlarm">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Alarm</string>
  </property>
  <widget class="QDialogButtonBox" name="alarmBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="TimeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>When will the alarm go off?</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>194</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="SoundLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Sound file:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browseButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="customButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>108</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Custom</string>
   </property>
   <attribute name="buttonGroup">
    <string>soundGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="defaultButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>108</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Default</string>
   </property>
   <attribute name="buttonGroup">
    <string>soundGroup</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>alarmBox</sender>
   <signal>accepted()</signal>
   <receiver>AddAlarm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>alarmBox</sender>
   <signal>rejected()</signal>
   <receiver>AddAlarm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="soundGroup"/>
 </buttongroups>
</ui>
